doc/api/cli.md

```diff
@@ -726,6 +726,17 @@ Export keyword before a key is ignored:
 export USERNAME="nodejs" # will result in `nodejs` as the value.
 ```
 
+### `--override-env-var`
```

Comment:
What about?

```suggestion
### `--override-env-file-vars`
```

Reply:
I'm not sure what is the best naming for here. @GeoffreyBooth wdyt?

Reply:
I think if we’re going to have a series of flags related to `--env-file`, they should all start with `--env-file`; similar to how all the test runner-related flags start with `--test`.

So this one could be `--env-file-override-local`.